website/routes.py

```diff
@@ -1166,6 +1166,16 @@ def make_url_map(app):
             json_renderer,
         ),
 
+        Rule(
+            [
+                '/project/<pid>/contributor/remove/',
+                '/project/<pid>/node/<nid>/contributor/remove/',
```

Comment:
I've reticketed deprecating the `node` URLs as [OSF-5518](https://openscience.atlassian.net/browse/OSF-5518).